### PR TITLE
[M1P-5] Update admin configs

### PR DESCRIPTION
**Pull request by adentong**
See https://github.com/BoltApp/bolt-magento1/pull/492.

#changelog Update the strings for Bolt's admin configuration and reorder and group them

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [ ] Test

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [x] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [x] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

**Comment by adentong** [comment]
Screen Shot 2020-03-31 at 5 03 02 PM
Screen Shot 2020-03-31 at 5 02 46 PM

**Comment by adentong** [comment]
Not sure why the Check Boltpay API button isn't showing up

**Comment by adentong** [comment]
Also, no migration needed since I made all the config paths the same as before.

**Comment by mccottry** [comment]
looking awesome so far!

**Comment by adentong** [comment]
@mccottry I still haven't been able to figure why the Check Boltpay API button isn't showing, and I don't think it's related to my changes here. Do you have any ideas?

**Comment by adentong** [comment]
The problem with the button not rendering turns out to be a setup issue, not a code issue.

**Comment by mccottry** [comment]
> @mccottry I still haven't been able to figure why the Check Boltpay API button isn't showing, and I don't think it's related to my changes here. Do you have any ideas?

I'll can take a look.

**Comment by adentong** [comment]
@mccottry Gah accidentally edited your comment. Sorry

**Comment by adentong** [comment]
But yea the PR is ready for review. The button not being rendered properly is due to a local misconfiguration.

File: app/code/community/Bolt/Boltpay/etc/config.xml [hunk 1]
```diff
@@ -269,41 +269,39 @@
...
+  .bolt-checkout-button.with-cards {
+    float: right;
+  }
+}
+        </selector_styles>
         <successpage>checkout/onepage/success</successpage>
+        <enable_merchant_scoped_account>0</enable_merchant_scoped_account>
         <allow_po_box>1</allow_po_box>
         <skip_payment>0</skip_payment>
-        <skip_confirm>1</skip_confirm>
```
Comment (matt-thomason) [inline app/code/community/Bolt/Boltpay/etc/config.xml:303]: I don't see `skip_confirm` added back in?

File: app/code/community/Bolt/Boltpay/etc/config.xml [hunk 2]
```diff
@@ -269,41 +269,39 @@
...
         <use_javascript_in_admin>0</use_javascript_in_admin>
-        <add_button_everywhere>0</add_button_everywhere>
-        <sort_order>0</sort_order>
```
Comment (matt-thomason) [inline app/code/community/Bolt/Boltpay/etc/config.xml:306]: Where do we render with `sort_order` removed?

File: app/code/community/Bolt/Boltpay/etc/config.xml (hunk 1 above)
Comment (adentong) [reply app/code/community/Bolt/Boltpay/etc/config.xml:303]: It's not used anywhere. Same with sort_order.

File: app/code/community/Bolt/Boltpay/etc/config.xml (hunk 2 above)
Comment (adentong) [reply app/code/community/Bolt/Boltpay/etc/config.xml:306]: sort_order is defined in system_xml. I don't know what this is doing here but I'm pretty convinced this doesn't do anything.

Comment (matt-thomason) [reply app/code/community/Bolt/Boltpay/etc/config.xml:306]: I'm pretty sure this effects the sort order of the payment methods on the page. But with a value of 0, I also don't think it's doing anything :D

**Comment by adentong** [comment]
@matt-thomason Gotcha. Will do.

**Pull request merged**